h5py/h5ds.pyx

```diff
@@ -76,6 +102,10 @@ def get_label(DatasetID dset not None, unsigned int idx):
 
 @with_phil
 def get_scale_name(DatasetID dscale not None):
+    """(DatasetID dscale) => STRING name_of_scale
+
+    Retrieves name of Dimension Scale dscale into buffer name.
```

Comment:
```suggestion
    Retrieves name of Dimension Scale dscale.
```

We wouldn't generally talk about reading into a buffer, unless the function takes the buffer as an input.